plugins/keymanagerng/app/javascript/widgets/app/components/secrets/secretDetails.jsx

```diff
@@ -164,9 +164,15 @@ const SecretDetails = () => {
         {secret?.isLoading && !secret?.data ? (
           <HintLoading />
         ) : secret?.isError ? (
-          <Message variant="danger">
-            {`${secret?.error?.statusCode}, ${secret?.error?.message}`}
-          </Message>
+          (!secret?.error?.statusCode && secret?.error?.message === '[object Object]') ? (
```

Comment:
is there no proper error message within `secret?.error?.message` to check for '[object Object]' looks weird ;-)

Reply:
oh never mind I read it in a wrong way